pages/en/advanced/mina-signer.mdx

```diff
@@ -20,35 +20,37 @@ Never reuse the same keypair from a testnet on mainnet!
 
 ## Installation
 
-The source code for our client SDK is open source and be seen [here](https://github.com/MinaProtocol/mina/tree/develop/frontend/client_sdk).
+The source code for our SDK is open source and be seen [here](https://github.com/MinaProtocol/mina/tree/develop/frontend/mina-signer).
```

Comment:
I'd suggest moving this above the `Installation` header, so people are less likely to get confused if this affects the installation step, but it still lets you accomplish making it prominent.